src/store.js

```diff
@@ -5,7 +5,7 @@ import { connectRouter, routerMiddleware } from 'connected-react-router';
 import { save, load } from 'redux-localstorage-simple';
 
 import config from '@plone/volto/registry';
-import reducers from '~/reducers';
+import reducers from '@plone/volto/reducers';
```

Comment:
instead of doing this i can also add an ``~/reducers`` alias in Mockup's webpack.

Reply:
@thet: I think you should add the alias is mockup. The problem is that Volto projects (which are the `~` root when Volto runs as a bootstrapped project) still have the reducers folder inside them. IMHO it would be better if we'd somehow get rid of that, too.